test/javascripts/turbolinks_test.coffee

```diff
@@ -16,6 +16,7 @@ describe 'Turbolinks', ->
     document.body.appendChild(createTurboNodule())
 
   afterEach ->
+    document.head.innerHTML = ""
```

Comment:
Rewriting these tests is absolutely a thing that needs to be done, but at the time we were planning to do it later, having deferred it in favour of shipping this by friday.

If we can't get it out by then I'm totally excited to make these less terrible :)